lib/console_web/controllers/router/device_controller.ex

```diff
@@ -172,19 +173,35 @@ defmodule ConsoleWeb.Router.DeviceController do
             Events.create_event(Map.put(event, "organization_id", organization.id))
           end)
           |> Ecto.Multi.run(:device, fn _repo, %{ event: event } ->
-            dc_used = 
+            dc_used =
               case event.sub_category in ["uplink_confirmed", "uplink_unconfirmed"] do
                 true -> event.data["dc"]["used"]
                 false -> 0
               end
+            packet_count = if dc_used == 0, do: 0, else: 1
+
             Devices.update_device(device, %{
               "last_connected" => event.reported_at_naive,
               "frame_up" => event.data["frame_up"],
               "frame_down" => event.data["frame_down"],
-              "total_packets" => device.total_packets + 1,
+              "total_packets" => device.total_packets + packet_count,
```

Comment:
I think we only want to count uplinks, im like 95% sure lol

Reply:
I think you right